rules_gapic/bazel/src/main/java/com/google/api/codegen/bazel/BUILD.bazel.raw_api.mustache

```diff
@@ -0,0 +1,169 @@
+# This file was automatically generated by BuildFileGenerator
+
+# This is an API workspace, having public visibility by default makes perfect sense.
+package(default_visibility = ["//visibility:public"])
+
+##############################################################################
+# Common
+##############################################################################
+load("@com_google_googleapis_imports//:imports.bzl", "proto_library_with_info")
```

Comment:
I think this is missing `load("@rules_proto//proto:defs.bzl", "proto_library")`

Reply:
You are right. It is optional now, but will be required for bazel 2.0 (that is why build, which used the gnerated files, did not fail). Added.